src/main/java/net/neoforged/neoforge/common/extensions/IFallableExtension.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.common.extensions;
+
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.entity.item.FallingBlockEntity;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Fallable;
+
+public interface IFallableExtension {
+    /**
+     * Called in {@link FallingBlockEntity#tick()} after vanilla processing on both server and client.
+     * <p>
+     * This is not called in the tick where the entity lands, see {@link Fallable}.
+     * 
+     * @param level           The current level.
+     * @param currentPosition The current position of the entity as a {@link BlockPos}.
+     * @param entity          The falling entity.
+     */
+    default void onStillFalling(Level level, BlockPos currentPosition, FallingBlockEntity entity) {}
```

Comment:
![image](https://github.com/user-attachments/assets/4103c79d-b68d-4596-bd9b-28bb2e66ceae)
Method would ideally be renamed before we merge

Reply:
I named it to match the vanilla onLand() in Fallable.

As a new name, if needed, I'd suggest fallingTick().